core/test_blockchain.go

```diff
@@ -1004,8 +1004,10 @@ func TestReorgReInsert(t *testing.T, create func(db ethdb.Database, chainConfig
 // Insert two different chains that result in the identical state root.
 // Once we accept one of the chains, we insert and accept A3 on top of the shared
 // state root
-//   G   (genesis)
-//  / \
+//
+//	 G   (genesis)
+//	/ \
+//
```

Comment:
could we maintain the diagram's ascii art alignment if we make this change?
(or yet better not modify the alignment in these comments)